RequestManagementSystem/scripts/dirac-rms-list-req-cache.py

```diff
@@ -0,0 +1,42 @@
+#!/bin/env python
+""" List the number of requests in the caches of all the ReqProxyies """
+__RCSID__ = "$Id: $"
+
+from DIRAC.Core.Base import Script
+Script.registerSwitch( '', 'Full', '   Print full list of requests' )
+Script.setUsageMessage( '\n'.join( [ __doc__,
+                                     'Usage:',
+                                     ' %s [option|cfgfile]' % Script.scriptName ] ) )
+
+if __name__ == "__main__":
+
+  from DIRAC.Core.Base.Script import parseCommandLine
+  parseCommandLine()
+
+  fullPrint = False
+
+  for switch in Script.getUnprocessedSwitches():
+    if switch[0] == 'Full':
+      fullPrint = True
+
+  import DIRAC
+
+  from DIRAC.RequestManagementSystem.Client.ReqClient import ReqClient
```

Comment:
Please put the imports at the beginning of the file https://www.python.org/dev/peps/pep-0008/#imports